osu.Game/Overlays/Settings/Sections/Gameplay/GeneralSettings.cs

```diff
@@ -34,8 +34,8 @@ private void load(OsuConfigManager config)
                 },
                 new SettingsCheckbox
                 {
-                    LabelText = "Lighten playfield during breaks",
-                    Bindable = config.GetBindable<bool>(OsuSetting.LightenDuringBreaks)
+                    LabelText = "Disable background effects during breaks",
```

Comment:
doesn't this sound wrong? lighten isn't lightening to 0, so there's still "effects" being applied.

Reply:
I'm not even sure how to name this better so that it's both accurate and understandable.

Reply:
Not 100% sure about having these both as one setting either, but if we are going to do it this way how about "Make background more visible during breaks"?